java/spark-2.0.0/utils200.scala

```diff
@@ -0,0 +1,19 @@
+package sparklyr
```

Comment:
We can call it `utils.scala` but it'd mean we can have a couple `utils.scala` tabs in RStudio. I actually prefer `utils.scala` for the reason you mentioned, and can change it back if the duplicate name in the code editor doesn't bother us.